decompiler/config/jak2/all-types.gc

```diff
@@ -8774,6 +8774,7 @@ Unlike [[engine]], users can use [[engine-pers]] as a parent class."
   (progress-graphics-ps2-lod-dist #x1337)
   (progress-graphics-msaa-off #x1338)
   (progress-graphics-display #x1339)
+  (language-name-finnish #x133a)
```

Comment:
you need to also add this to `decompiler/config/jak2/all-types.gc` and the failing builds will pass.